language/go/testdata/protos_explicit_default/BUILD.want

```diff
@@ -0,0 +1,33 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
```

Comment:
Couple more cases to test:

* Setting a directive to the empty string in a subdirectory goes back to the default behavior in that subdirectory.
* Setting a directive with multiple labels separated by commas works.

Reply:
Added both.